### DISCUSSION TITLE
Shutdown problem

### DISCUSSION THREAD

**Discussion by ramdani1978** [valnoxy/t480-oc#58] [Q&A] [answered]
Hello valnoxy,

Thank for your sharing and I've been using your EFI for Ventura, but I experience shutdown problem.
Every time I press shutdown, the laptop cannot shutdown, instead it restart again and when login showing error.

Are experiencing this problem also?
Besides that can you share the BIOS setting also?

Thank you for your support..

**Comment by BanannaFannaFoFax** [comment]
I have that exact same issue too.

**Reply by nicosrs** [reply]
i have power on ac on bios, and i have this issue.
when i press shutdown i turn also off the power to stop restarting

**Reply by rivalscott04** [reply]
i use this efi on monterey and everythinng worked, except some noise in 3.5 jack port

**Reply by Easyinspace32** [reply]
> I have that exact same issue too.

Hello it's same for me. when I shutdown a laptop, the laptop restart... I think missing:
DORTANIA GUIDE:
For this we need the following:
FixShutdown-USB-SSDT.dsl (opens new window)
_PTS to ZPTS Patch (opens new window)
USB Controller's ACPI Path

**Comment by cchewanik** [comment, marked as answer]
What spec is your T480? I have a no touch screen version, with no external GPU and have no issues. The EFI Valnoxy put together includes kexts for touch screen (VoodooInput) which made my system unstable, I removed them and have had no issues.

**Reply by ramdani1978** [reply]
My T480 is touch screen version.
I will try to remove voodoo input for touchscreen, which kext is it? I have 5 kext file start with voodoo?

**Reply by cchewanik** [reply]
I kept voodoops2controller and voodoosmbus

**Reply by ramdani1978** [reply]
It works, after removing other voodoo and kept only voodoops2 and voodoosmbus.
Thank you [cchewanik](https://github.com/cchewanik)

**Comment by angelaguayo** [comment]
can someone solves that? i'm in the same situation

**Comment by valnoxy** [comment]
Hi! 👋

as [cchewanik](https://github.com/cchewanik) already said, it is a issue with VoodooI2C and Ventura. I have already disabled it by default in the development branch and will be rolled out soon as version 1.3.0.
